charts/ti-key-release-2/templates/jss-server.yaml

```diff
@@ -87,3 +107,15 @@ spec:
            path: /var/tsi-secure
            # directory must exist
            type: Directory
+{{- if eq .Values.jssService.type "vtpm2-server" }}
```

Comment:
there looks like a discrepency in this and the above if statement if the jssService type is `vtpm2-server`, then the volume mount would exists without the volume? This would cause the container to fail i believe

Reply:
In both cases, for volume-mount and volume, they are only created when all three nested conditions are satisfied: 
```
{{- if eq .Values.jssService.type "vtpm2-server" }}
     {{- if .Values.jssService.tpm }}
          {{- if eq .Values.jssService.tpm.interface_type "dev" }}
```